dataland-backend/src/main/kotlin/org/dataland/datalandbackend/services/DataMetaInformationManager.kt

```diff
@@ -21,27 +20,13 @@ class DataMetaInformationManager(
 
     /**
      * Method to associate data information with a specific company
-     * @param company The company to associate the data meta information with
-     * @param dataId The id of the dataset to associate with the company
-     * @param dataType The dataType of the dataId
+     * @param updatedMetaData The data meta information which should be stored
      */
     @Transactional
     fun storeDataMetaInformation(
-        dataId: String,
-        dataType: DataType,
-        uploaderUserId: String,
-        uploadTime: Long,
-        company: StoredCompanyEntity,
+        updatedMetaData: DataMetaInformationEntity,
```

Comment:
why `updatedMetaData`?

Reply:
Is this only about the naming or is it about why the input as a whole got changed?
Namingwise my thought now would be that the name is not appropiate and would suggest to change it to "dataMetaInformation".

Reply:
as far as I see, this method is also used to store the initial meta data - not only the updated.
Hence:
`fun storeDataMetaInformation(dataMetaInformation DataMetaInfromationEntity)`

Reply:
Done

Reply:
ok